src/cards/katyaCard/katyaCard.jsx

```diff
@@ -4,22 +4,23 @@ import image from './KatyaHeadshot.jpg';
 
 function KatyaCard() {
   return (
-    <div className="cardContainer">
+    <div className="kat-cardContainer">
 
-        <div className="card">
-            <img className="imageContainer" src={image} alt="example headshot of a woman" />
+        <div className="kat-card">
+            <img className="kat-imageContainer" src={image} alt="example headshot of a woman" />
+            <div class="kat-image-border"></div>
```

Comment:
because it's JSX, you need to update all of the "class" tags to "className".

I'm actually surprised everything rendered right.... you card looks AMAZING!

Reply:
Oooh okay got it! I'll do that now :) and thank you!!

Reply:
Done! :)